### ISSUE TITLE
Nothing displayed in IE11

### ISSUE THREAD

**Issue by traxxion** [ChristophWurst/sketch#36]
Sketch app loads but an otherwise blank screen
Debugger enabled shows:

Line: 140
Error: Script error for: start
http://requirejs.org/docs/errors.html#scripterror

**Comment by ChristophWurst** [comment]
Hi @traxxion,
I gave it a quick test and could not reproduce the error:
![bildschirmfoto von 2016-01-27 15-32-29](https://cloud.githubusercontent.com/assets/1374172/12616402/439cf0e0-c50b-11e5-9b2a-dd451b0dda60.png)
The app loads as expected.

Which exact version are you using? The one from the appstore?

**Comment by traxxion** [comment]
Thanks for checking. The version I have installed is 0.1.1 - I believe that this version was downloaded from the app store. As far as I can see there is nothing in the owncloud log relating to Sketch. I have not had a chance to check the web server logs, but I will try to do so this weekend. I am running owncloud 8.2.2 and of course the issue may not be related to IE11 specifically, but that is the main browser that I use at work, etc. However, I can't remember if I was initially testing it in Epiphany on my laptop at home. I have just switched Linux distros and the default is Firefox, so I will try and get some more information to troubleshoot with. Thanks.
